zerver/tests/test_realm.py

```diff
@@ -179,10 +179,10 @@ def test_do_deactivate_realm_clears_user_realm_cache(self) -> None:
         Hamlet, and we end by checking the cache to ensure that his
         realm appears to be deactivated.  You can make this test fail
         by disabling cache.flush_realm()."""
-        self.example_user("hamlet")
+        get_user_profile_by_email("hamlet@zulip.com")
         realm = get_realm("zulip")
         do_deactivate_realm(realm)
-        user = self.example_user("hamlet")
+        user = get_user_profile_by_email("hamlet@zulip.com")
```

Comment:
Yeah, this looks to have been incorrectly ported to `example_user` in some sweep.